src/webgpu/api/operation/rendering/depth.spec.ts

```diff
@@ -20,8 +159,62 @@ g.test('depth_bias')
 g.test('depth_disabled').desc(`Tests render results with depth test disabled`).unimplemented();
 
 g.test('depth_write_disabled')
-  .desc(`Tests render results with depth write disabled`)
-  .unimplemented();
+  .desc(
+    `
+  Test that disabling depth writes works and leaves the depth buffer unchanged.
+  `
+  )
```

Comment:
Just as a small control case, let's add a case with depth write disabled, that expects the result to be red (because the last draw fails the depth test).

Reply:
ok, then let me add a case that expects the result to be red because it passes the depth test.

Reply:
Sorry, I made a typo, I meant we should have another case with depth write enabled. I'll make a PR